### DISCUSSION TITLE
[5.x] Allow Duplication of Entry Types

### DISCUSSION THREAD

**Discussion by DowleyDeveloped** [craftcms/cms#15977] [Ideas] [answered]
With the merge of Matrix Blocks into Entry Types (Loving it btw!), it would be really handy if we could duplicate entry types themselves. 

We have a few components which have the same fields (layout etc) and it would be much easier to just duplicate existing types instead of build from scratch each time.

**Comment by piotrpog** [comment]
Yeah, just add "save as new entry type", just like we can with cloning regular entries. Would be pretty useful.

**Reply by brandonkelly** [reply, marked as answer]
Added this for Craft 5.8 (691bc20e0e907121b8bc9045f16dece613ed25c0).

**Reply by alexanderbuergin** [reply]
Love that one! Thanks.
